dev/reformat

```diff
@@ -20,6 +20,14 @@ set -x
 
 KYUUBI_HOME="$(cd "`dirname "$0"`/.."; pwd)"
 
-PROFILES="-Pflink-provided,hive-provided,spark-provided,spark-block-cleaner,spark-3.3,spark-3.2,spark-3.1,tpcds"
+# python style checks rely on `black` in path
+if ! command -v black &> /dev/null
+then
+    SPOTLESS_PYTHON_INCLUDES=""
+else
+    SPOTLESS_PYTHON_INCLUDES=",spotless-python"
+fi
+
+PROFILES="-Pflink-provided,hive-provided,spark-provided,spark-block-cleaner,spark-3.3,spark-3.2,spark-3.1,tpcds${SPOTLESS_PYTHON_INCLUDES}"
```

Comment:
```suggestion
PROFILES="-Pflink-provided,hive-provided,spark-provided,spark-block-cleaner,spark-3.3,spark-3.2,spark-3.1,tpcds"

# python style checks rely on `black` in path
if ! command -v black &> /dev/null
then
    echo "Skip Python lint since `black` is not available."
else
    PROFILES="${PROFILES},spotless-python"
fi
```